genai-perf/README.md

```diff
@@ -105,7 +105,9 @@ You can also build Perf Analyzer [from source](../docs/install.md#build-from-sou
 ### Install GenAI-Perf from source
 
 ```bash
-pip install git+https://github.com/triton-inference-server/perf_analyzer.git#subdirectory=genai-perf
+git clone https://github.com/triton-inference-server/perf_analyzer.git && cd perf_analyzer
+
+pip install -e genai-perf
```

Comment:
why did we split the single command to multiple commands?

Reply:
what do you mean?

Reply:
It was `pip install git+https://github.com/triton-inference-server/perf_analyzer.git#subdirectory=genai-perf` before.

Reply:
now it's 
```
git clone https://github.com/triton-inference-server/perf_analyzer.git && cd perf_analyzer
pip install -e genai-perf
```

Reply:
Its possible that it was a discrepancy with the template vs the readme.
we can revert it if you prefer

Reply:
yes, we want a single pip install command for this. We don't want the users to go through two steps of cloning the repo and then pip install it.

Reply:
Reverted